tests/mappings/test_mappings.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+"""Test mappings."""
```

Comment:
Have we not agreed on using pytest for making tests?

Reply:
I think we should use pytest when it make sense to do so. For simple cases pytest just make the tests more difficult to understand and makes it impossible to use the tests for interactive development and debugging. The latter greatly improves my development efficiency of good code. 

When running the test scripts from pytest, we still get most of the advantages of pytest, autodetection of tests, running several tests in parallell, skipping selected tests, etc...

However, I still think pytest may be the right tool for complex cases where test separation is important (even though that can be achieved by splitting the tests into several files) when we need parametrised tests, or similar. 

But in this case, I think a test script is superior.

Reply:
Ok, we really need to decide on a team level what we go for, but I will approve this for now.